.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,24 @@
+# Issue Description
+  [Type of Issue]
+
+## Problem
+  [Describe the problem we are trying to solve.]
+
+## Success
+  [What should this task accomplish when it's considered completed.]
+
+### Requirements
+
+- [ ] Add
```

Comment:
@karlk08 are these spaces correct? I cannot tell? It looks like dash space but I think they need to be together

Reply:
I removed the space in between dash and square brackets.